data-es2016plus.js

```diff
@@ -4591,6 +4591,26 @@ exports.tests = [
           hermes0_7_0: true,
         }
       },
+      {
+        name: 'spread parameters after optional chaining',
+        exec: function(){/*
+          var fn = null;
+          var n = null;
+          var o = {};
+
+          return fn?.(...[], 1) === void undefined && fn?.(...[], ...[]) === void undefined && o.method?.(...[], 1) === void undefined && n?.method(...[], 1) === void undefined;
+        */},
+        res : {
+          ie11: false,
+          firefox10: false,
+          firefox52: false,
+          firefox73: false,
+          firefox74: true,
+          chrome89: false,
```

Comment:
is there a version of chrome where this ever worked?

Reply:
I downloaded an old Chrome 80 build, and it failed there, too. I don't think it's ever been supported.

Reply:
I'll add this so that older versions are marked as "no" and not as "unknown"